gulpfile.js

```diff
@@ -8,7 +8,27 @@
    gulp/tasks/default.js specifies the default set of tasks to run
    when you run `gulp`. */
 
+const gulp = require( 'gulp' );
 const requireDir = require( 'require-dir' );
 
 // Require all tasks in gulp/tasks, including subfolders.
 requireDir( './gulp/tasks', { recurse: true } );
+
+gulp.task( 'build',
+  gulp.series(
+    gulp.parallel(
+      'styles',
+      'scripts',
+      'images'
+    ),
+    'copy'
+  )
+);
+
+gulp.task( 'default',
+  gulp.series(
+    'build',
+    'lint:scripts',
+    'test:unit'
+  )
```

Comment:
Is there a reason these need to be in series? Also, for what it's worth, I was able to successfully test putting `build` back to the end in my local environment.

Reply:
Yeah, it would be nice if these ran in parallel.

Reply:
Updated to parallel!